### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by vipulchakravarthy**
I have completed task-1 & 2. Pls, review it.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -45,10 +50,22 @@ def authentication():
...
+
+    # Forget any user ID
+    session.clear()
+
+    # Redirect user to login form
+    return redirect("/register")
+
 @app.route("/admin", methods=["GET"])
 def fetch_users():
     data = User.query.all()
```
Comment (praveengarimella) [inline project1/application.py]: desc order?

Comment (vipulchakravarthy) [reply project1/application.py]: Okay, I'll update the query.

**Review by praveengarimella — CHANGES_REQUESTED** [review]
I don't want to repeat the same comments again. Please remove the bin files from the repo and then I'll review the code.

**Comment by vipulchakravarthy** [comment]
Sorry, It took some time for me to figure out the issue with .gitignore. I pushed all the binary files into .gitignore.

File: project1/application.py (hunk 1 above)
Comment (vipulchakravarthy) [reply project1/application.py]: I have updated the query.

**Comment by vipulchakravarthy** [comment]
Can I merge this branch?

**Comment by praveengarimella** [comment]
Does this still need a review? If not please close it. @vipulchakravarthy

**Pull request merged**